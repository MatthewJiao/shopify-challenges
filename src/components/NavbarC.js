import React, { useState } from 'react';
import Container from 'react-bootstrap/Container';
import Navbar from 'react-bootstrap/Container';

const NavbarC = () => {
  

  return (
    
        <div className = "pb-3 pt-1" style = {navStyle}>
           Image Repository
        </div>
    
  )
}

const navStyle = {
    color: "#c5d4d2",
    backgroundColor: "#deffe2",
    opacity: 0.95,
    position: "fixed",
    top: 0,
    fontWeight: "bold",
    fontSize: "2rem",
    width: "100%",
    display: "flex",

    justifyContent: "center",
}

const navImageStyle = {
    height: "2.5rem", 
    paddingTop: "0.2rem",
    transform: "rotate(20deg)"
}


export default NavbarC;