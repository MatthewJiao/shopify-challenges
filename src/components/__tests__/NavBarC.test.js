import React from 'react'
import reactDom from 'react-dom'
import { cleanup, render } from '@testing-library/react'
import renderer from 'react-test-renderer'

import NavbarC from '../NavbarC'


afterEach(cleanup)
it("Navbar renders properly and without issue", () => {
    const div = document.createElement("div");
    reactDom.render(<NavbarC />, div);
})


it("matches snapshot", () => {
    const tree = renderer.create(<NavbarC />).toJSON();
    expect(tree).toMatchSnapshot()
})
