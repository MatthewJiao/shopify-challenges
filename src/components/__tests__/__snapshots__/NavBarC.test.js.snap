// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="pb-3 pt-1"
  style={
    Object {
      "backgroundColor": "#deffe2",
      "color": "#c5d4d2",
      "display": "flex",
      "fontSize": "2rem",
      "fontWeight": "bold",
      "justifyContent": "center",
      "opacity": 0.95,
      "position": "fixed",
      "top": 0,
      "width": "100%",
    }
  }
>
  Image Repository
</div>
`;
